src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.cs

```diff
@@ -1324,35 +1324,15 @@ private DebugId GetLambdaId(SyntaxNode syntax, ClosureKind closureKind, int clos
             if (_analysis.LambdaScopes.TryGetValue(node.Symbol, out lambdaScope))
             {
                 containerAsFrame = _frames[lambdaScope];
-                var structClosureParamBuilder = ArrayBuilder<TypeSymbol>.GetInstance();
-                while (containerAsFrame != null && containerAsFrame.IsValueType)
+                structClosures = _analysis.CantTakeRefParameters(node.Symbol)
+                    ? ImmutableArray<TypeSymbol>.Empty
```

Comment:
>ImmutableArray<TypeSymbol>.Empty [](start = 22, length = 32)

It looks like other branches assign ```default(ImmutableArray<TypeSymbol>)```. Is ```ImmutableArray<TypeSymbol>.Empty``` used to communicate different information? Consider standardizing on the same value otherwise.